app/views/root/_option_report_child_abuse.html.erb

```diff
@@ -0,0 +1,43 @@
+<% places ||= [] %>
+<% places.each do |place| %>
+  <li>
+    <div class="place group">
+      <div class="information">
+        <div class="address vcard">
+          <div class="adr org fn">
+
+            <p class="adr">
+              <% if place["name"].present? %>
+                <span class="fn">You can call the children's social care team at <%= place["name"] %> Council</span>
```

Comment:
Would it be better to leave "Council" off here and make sure it's included in the names of all the places in the data-set?  This allows for the names to be the full council name and include their own preferred way of naming themselves.  Also means we don't have to have the name be "Blahville District" or "Areaton Borough" if that part of the name is important.